src/LanguageSwitcher/style.js

```diff
@@ -0,0 +1,28 @@
+import styled from '@emotion/styled'
+import { theme } from '~/utils/styles'
+
+export const LanguageSwitcherContainer = styled.div`
+  display: inline-flex;
+
+  p {
+    margin-bottom: 0px !important;
+  }
+
+  p.current {
+    font-weight: bold;
+  }
+
+  .Header__LanguageSwitcher-button {
+    text-decoration: none;
+    color: black;
+    font-family: ${theme.fontFamily.secondary};
+    line-height: 26px;
+    font-style: normal;
+    font-size: ${theme.fonts.heading6};
+
+    @media (min-width: ${theme.breakpoints.md}) {
```

Comment:
to call the theme object, do as such: `@media (min-width: ${props => props.theme.breakpoints.md})`